atlp-rwanda/Team-furebo-E-commerce-fn
# Ft: admin update/buyer track order #185172207

**What does this PR do?**
This PR adds functionality to track the status of an order for buyers.

**Description of Task to be completed**
The task involves implementing the following features:

Order history page: Display a list of the buyer's previous orders, including order number, total cost, and expected delivery date.
Order detail page: Display detailed information about a specific order, including order status, items purchased, delivery information.
Real-time order status updates: Implement real-time updates for the order status using web sockets or a similar technology.
Contact customer support button: Add a prominent button on the order detail page to direct the buyer to the customer support page.

**How should this be manually tested?**

Access the order history page and verify that it displays a list of previous orders with the required information.
Click on an order from the list and ensure that it navigates to the order detail page.
On the order detail page, refresh the page and verify that the order status updates in real-time.
Click on the contact customer support button and confirm that it redirects to the customer support page.

**What are the relevant pivotal tracker/Trello stories?**
#185172207
